src/Packagist/WebBundle/Resources/config/algolia_settings.yml

```diff
@@ -1,22 +1,22 @@
 searchableAttributes:
-    - "name"
-    - "package_name"
-    - "package_organisation"
-    - "description"
-    - "tags"
+  - "unordered(package_name)"
+  - "unordered(tags)"
+  - "unordered(package_organisation)"
+  - "unordered(description)"
```

Comment:
I had to append `unordered(name)` here otherwise a search for an exact package name doesn't even match, which sounds insane to me.. say a search for e.g. symfony/monolog-bundle should return that one package at the very least.